src/sentry/integrations/tasks/sync_status_inbound.py

```diff
@@ -294,3 +305,25 @@ def sync_status_inbound(
             activity_type=ActivityType.SET_UNRESOLVED,
             activity_data=activity_data,
         )
+
+        for group in affected_groups:
+            issue_unresolved.send_robust(
+                project=group.project,
+                user=default_user,
+                group=group,
+                transition_type=provider.key,
+                sender=f"unresolved_with_{provider.key}",
+            )
+
+
+def _get_default_user_info(organization: Organization) -> tuple[int, RpcUser]:
+    try:
+        default_user_id = organization.default_owner_id
+    except IndexError:
+        logger.exception("Error getting default user")
+        default_user_id = "<unknown>"
+
+    default_user = user_service.get_user(default_user_id)
+    assert default_user is not None, "unable to fetch user"
```

Comment:
Should we be catching this `AssertionError` or is it intended to cause a failure?

Reply:
uhhhhh, lemme think. If we do not have a user, we won't be able to attach a user to the dispatched signal and the analytics. I think we can fail silently instead then ? Alternatively we could send the webhooks with no user id, but that seems sus 🤔

Reply:
So looking at it again, the `default_user` here was [added to satiate a metric](https://github.com/getsentry/sentry/issues/87042). I would actually use no `user` here since we're resolving the issue from an external provider. 

In this case, using the organization owner as a default is actually more incorrect, and probably unexpected. I'd just leave it empty, since the resolution indicates the provider anyway.

Reply:
Ah nice find, yeah I wonder if we can change the flow to put like Sentry as the user or just handle the `None` case in the receivers 🤔

Reply:
Ah I went looking manually for what functions are connected to `issue_resolved` & `issue_unresolved`, and it looks like t[hat's what we do for webhooks ](https://github.com/getsentry/sentry/blob/crl/signal-workflow-on-integration-upd/src/sentry/sentry_apps/api/serializers/app_platform_event.py#L20-L32)(say the user is Sentry, if the given user/actor is `None`). For other places [we get the default org owner again if given user is None](https://github.com/getsentry/sentry/blob/crl/signal-workflow-on-integration-upd/src/sentry/receivers/features.py#L226-L230) which is not ideal 🫠 . 

I dont want to add more changes unrelated to this PR, so I can make another PR to change the other places lookups to be "<unknown>"